### PR TITLE
Prepare release 2.1.0

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,17 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 2.0.2 - [XXXX-XX-XX]
+## 2.1.1 - [XXXX-XX-XX]
```
Comment (Lucpen) [inline CHANGELOG.md:6]:
```suggestion
## X.X.X - [XXXX-XX-XX]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,17 @@
...
+
+### `Added`
+
+### `Fixed`
+
+### `Parameters`
+
+### `Changed`
+
+## 2.1.0 - Elve [2024-06-26]
```
Comment (Lucpen) [inline CHANGELOG.md:16]:
```suggestion
## 2.1.0 - Elf [2024-06-26]
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed DROP output column names to camel case [#132](https://github.com/genomic-medicine-sweden/tomte/pull/132)
 - Updated bcftools view and norm to be able to create index within the actual module [#137](https://github.com/genomic-medicine-sweden/tomte/pull/137)
 - Updated bcftools annot to be able to create index within the actual module [#140](https://github.com/genomic-medicine-sweden/tomte/pull/140)
+- Updated modules bcftools/merge, bcftools/mpileup, bcftools/stats, ensemblvep/vep, fastp, gawk, multiqc, salmon/quant, samtools/faidx, samtools/index and samtools/view [#141](https://github.com/genomic-medicine-sweden/tomte/pull/141)
```
Comment (fellen31) [inline CHANGELOG.md:41]: I see raredisease has a "Module updates" table in the CHANGELOG, which looks pretty nice imo, like the parameters table you had in the previous release.  

https://github.com/nf-core/raredisease/blob/master/CHANGELOG.md#module-updates

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,17 @@
...
+## X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+### `Parameters`
+
+### `Changed`
+
```
Comment (fellen31) [inline CHANGELOG.md:15]: Not sure it actually matters, but maybe this part should be added _after_ the release is done.

**Review by fellen31 — APPROVED** [review]
Looks good! Two comments about the CHANGELOG.

File: CHANGELOG.md (hunk 3 above)
Comment (Lucpen) [reply CHANGELOG.md:41]: I noticed in on your PR, it does look super nice, I will add it before merging

File: CHANGELOG.md (hunk 4 above)
Comment (Lucpen) [reply CHANGELOG.md:15]: That's true, I will remove it and read it later

**Pull request merged**